src/gui_tool/gui/bb_gui.py

```diff
@@ -39,6 +40,10 @@
     ["t", "Opens window for user customizable tags"],
 ]
 
+BB_CLASS_DEFAULT_OPTIONS = [
+    "Car", "Truck", "Motorcycle", "Van", "Pedestrian"
```

Comment:
we have been using all lower case for the options so I think these should be the same